mne_bids/meg_bids.py

```diff
@@ -315,9 +317,17 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
     data_meta_fname = make_bids_filename(
         subject=subject_id, session=session_id,
         suffix='%s.json' % kind, prefix=data_path)
-    raw_file_bids = make_bids_filename(
-        subject=subject_id, session=session_id, task=task, run=run,
-        suffix='%s%s' % (kind, ext), prefix=data_path)
+    if ext in ['.fif', '.gz', '.ds']:
```

Comment:
I'm not sure I get this branching. 

Also, `.gz` is now being removed from fif specification. You can remove it if you like.

Reply:
I did it because those file types are put in the same folder as the sidecar json and other files, while any other raw files will go in a sub-directory...